### DISCUSSION TITLE
[A-L] (16 Gennaio 2023) Esercizio 1

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#364] [Esami A-L]
Un testo d’esame comprende 5 domande di Combinatoria, 10 domande di Logica e 5 domande di Algebra.

1. In quanti modi posso scegliere 10 domande?

2. In quanti modi posso scegliere 10 domande di cui esattamente 4 di Logica?

3. In quanti modi posso scegliere 10 domande di cui almeno una di Combinatoria e almeno una di Logica?

**Comment by FeddyLix17** [comment]
Rappresentando l'insieme delle domande come

$$
    \Large D = \\\{ c_1,\ c_2,\ c_3,\ c_4,\ c_5,\ l_1,\ l_2,\ l_3,\ l_4,\ l_5,\ l_6,\ l_7,\ l_8,\ l_9,\ l_{10},\ a_1,\ a_2,\ a_3,\ a_4,\ a_5\\\}
$$ 

(dove c = Combinatoria, l = Logica e a = Algebra)

1. $\Large \binom{20}{10}$

---

2. $\Large \binom{10}{4} * \binom{10}{6}$

---

3. $\Large  \binom{20}{10} - \left[\binom{15}{10} + \binom{10}{10}\right] = \binom{20}{10} - \left[\binom{15}{10} + 1 \right]$

**Reply by Dieal** [reply]
@FeddyLix17 
Non sono d'accordo sul punto 3.
La soluzione che propongo è per passaggio al complemento:
TOT - [Domande senza logica] - [Domande senza combinatoria] = (20 10) - (10 10) - (15 10)

**Reply by BenjiKunji** [reply]
@FeddyLix17 
per il secondo punto non dovrebbe essere (10 4) * (10 6)?
